Makefile

```diff
@@ -60,13 +61,24 @@ debug:
 clean:
 	find . -name '*~' -exec rm {} \;
 
+################################################################
+## Local mail server
+
+LOCALMAIL_PID=twistd.pid
+LOCALMAIL_MBOX=/tmp/localmail.mbox
+kill-local-mail:
+	if [ -r $(LOCALMAIL_PID) ]; then echo kill -9 `cat $(LOCALMAIL_PID)` ; kill -9 `cat $(LOCALMAIL_PID)` ; /bin/rm -f $(LOCALMAIL_PID) ; fi
+	/bin/rm -f $(LOCALMAIL_MBOX)
```

Comment:
I saw this line return non-zero and fail the script on a test run. Maybe 

/bin/rm -f $(LOCALMAIL_MBOX) || true

instead

Reply:
Odd; `rm -f should` never generate an error...

Reply:
I think that your comments were from a different version? The current run doesn't have an error here.